pkg/sloop/server/internal/config/config.go

```diff
@@ -25,57 +25,57 @@ import (
 const sloopConfigEnvVar = "SLOOP_CONFIG"
 
 type SloopConfig struct {
-	// These fields can only come from command line
-	ConfigFile string
+	// These fields can only come from command line or flag args
+	ConfigFile string	`json:"config"`
 	// These fields can only come from file because they use complex types
 	LeftBarLinks  []webserver.LinkTemplate         `json:"leftBarLinks"`
 	ResourceLinks []webserver.ResourceLinkTemplate `json:"resourceLinks"`
 	// Normal fields that can come from file or cmd line
-	DisableKubeWatcher       bool          `json:"disableKubeWatch"`
-	KubeWatchResyncInterval  time.Duration `json:"kubeWatchResyncInterval"`
-	WebFilesPath             string        `json:"webfilesPath"`
-	BindAddress              string        `json:"bindAddress"`
+	DisableKubeWatcher       bool          `json:"disable-kube-watch"`
```

Comment:
does this require changes to helm charts (any place configmaps are stored)?

Reply:
Yes, you would have to update the config json file with these tags(if you are specifying a config file).